etc/test_itemized.out

```diff
@@ -6,5 +6,5 @@ TEST writeback_success noWriteback RESULT fail writeback writeback_success ALPHA
 TEST writeback_success noPointState RESULT fail writeback writeback_success ALPHA 5 timeout waiting for point filter_differential_pressure_setpoint to have value_state applied
 TEST pointset_sample_rate fixedSampleRate=10 RESULT fail pointset pointset_sample_rate BETA 5 Failed check that time period between successive pointset events is between 1 and 5 seconds
 TEST system_mode_restart noPersist RESULT fail system.mode system_mode_restart ALPHA 5 Failed check that restart count increased by one
-TEST broken_config barfConfig RESULT fail system broken_config ALPHA 5 timeout waiting for log category `system.config.receive` level `DEBUG` was logged
+TEST broken_config barfConfig RESULT fail system broken_config ALPHA 5 timeout waiting for minimum test duration of 30s
```

Comment:
I don't understand this failure case? `timeout waiting for minimum test duration`

Reply:
Well, I mean because it's nonsensical!  Thank you for calling me out on my shenanigans. I've cleaned it up so it reports the original message (I'll spare you the details of _why_ it changed!)